python/lsst/dax/obscore/obscore_exporter.py

```diff
@@ -342,7 +457,7 @@ def _make_record_batches(self, batch_size: int = 10_000) -> Iterator[RecordBatch
 
         collections: Any = self.config.collections
         if not collections:
-            collections = ...
+            collections = "*"
```

Comment:
Hmm, does the new query system support globs for collections? There could be some confusion here too, as obscore config says that collection names can be regexes, and regexes do not work with the new system. We'll need to do something about it.

Reply:
It definitely supports globs. Sounds like we may have to change obscore config to use globs as well.

Reply:
Actually I was confused. Clearly the low-level system doesn't support wildcards so I raise here now.